data/strings/strings.txt

```diff
@@ -28978,7 +28978,7 @@
     eu = Zure dohaintza eta laguntzarekin, Munduko mapa onenak sor ditzakegu!
     fa = با کمک های مالی و حمایت شما، ما می توانیم بهترین نقشه های جهان را ایجاد کنیم!
     fi = Lahjoituksellasi ja tuellasi voimme luoda maailman parhaat kartat!
-    fr = Avec vos dons et votre soutien, nous pouvons créer les meilleures cartes du monde !
+    fr = Avec vos dons et votre soutien, nous pouvons créer les meilleures cartes du monde !
```

Comment:
Why is this change here? (and further)
It looks like there was a non-breakable space, which makes sense